CHANGELOG.md

```diff
@@ -13,7 +12,7 @@ x.x.x Release notes (yyyy-MM-dd)
 * File format: Generates Realms with format v9 (Reads and upgrades all previous formats)
 
 ### Internal
-* None.
+* Added a couple of options to `Realm.Session.config` to ease debugging.
```

Comment:
which couple of options are referred here? And how they ease debugging.

Reply:
They are not really useful (I don't use them in any test).

Reply:
Let me write it more precisely.